item.rb

```diff
@@ -0,0 +1,38 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :archived, :id, :name
+  attr_reader :label, :genre, :author
+
+  def initialize(id, name, publish_date, archived = false)
+    @id = id || Random.rand(1..100)
+    @name = name
+    @archived = archived
+    @publish_date = Date.parse(publish_date)
+  end
+
+  def can_be_archived?
+    current_date = Date.today
+    time_diff = current_date.year - @publish_date.year
+    time_diff >= 10
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
```

Comment:
- [ ] I would recommend adding specs for the `item` class as well as there are methods like `can_be_archived?` and `move_to_archive` that need to be tested.